tool/tctl/common/collection.go

```diff
@@ -1138,6 +1139,30 @@ func (c *samlIdPServiceProviderCollection) writeText(w io.Writer, verbose bool)
 	return trace.Wrap(err)
 }
 
+type botCollection struct {
+	bots []*machineidv1pb.Bot
+}
+
+func (c *botCollection) resources() []types.Resource {
+	resources := make([]types.Resource, len(c.bots))
+	for i, b := range c.bots {
+		resources[i] = types.Resource153ToLegacy(b)
+	}
+	return resources
+}
+
+func (c *botCollection) writeText(w io.Writer, verbose bool) error {
```

Comment:
`verbose` is unused

Reply:
The signature of this method must include the `verbose` parameter in order to match the `resourceCollection` interface.